templates/zerver/portico-header.html

```diff
@@ -1,5 +1,8 @@
 <div class="header portico-header">
     <div class="header-main" id="top_navbar">
+        {% if page_is_help_center or page_is_api_center %}
+        <a class="skip-link " href="#skip-link-target" tabindex="0">Skip To Main Content</a>
```

Comment:
[Zulip uses sentence-case](https://zulip.readthedocs.io/en/latest/translating/translating.html#capitalization), not title-case.

Reply:
And this should be tagged for translation; had you done, the linters would have caught the case issue.

https://zulip.readthedocs.io/en/latest/translating/internationalization.html